arrows/klv/klv_0601.cxx

```diff
@@ -17,6 +17,7 @@
 #include "klv_series.hpp"
```

Comment:
Minor nit-pick. Should we switch these includes from quotes to brackets like you did for klv_0806.cxx for consistency?  i.e. `"klv_length_value.h"` -> `<arrows/klv/klv_length_value.h>`.